### PR TITLE
Add collection entry for road and  building detection data

### PR DESCRIPTION
**Pull request by Dorothyrono**
Add new data product + sandbox data pages for Roads & Buildings

### CODE REVIEW

**Review by zansinergise — CHANGES_REQUESTED** [review]
What's the status with the map - are the polygons being created?

Note: I would wait with merging this until we get a final comment regarding the colors which should be used in the evalscripts, since you will likely have to change the highlight photos

Otherwise looks good 👍

File: collections/road-and-building-detection.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        Values: "0-255, (the greater the pixel value, the greater the likelihood that the pixel is classified correctly)"
+      - Name: buildings
+        Description: "building detection"
+        Values: "0-255, (the greater the pixel value, the greater the likelihood that the pixel is classified correctly)"
+      - Name: confidence
+        Description: "confidence"
+        Values: "0-255"
+      - Name: alpha
+        Description: "alpha"   
+        Values: " "
```
[the remark this answers is not in the document]
Comment (Dorothyrono) [reply collections/road-and-building-detection.yaml:37]: > What's the status with the map - are the polygons being created?

expected today.  
Agree,  will wait with the merging

**Comment by zcernigoj** [comment]
> What's the status with the map - are the polygons being created?

They are now.

File: collections/road-and-building-detection.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (zansinergise) [reply collections/road-and-building-detection.yaml:37]: Looks like the colors will not change, so all good from my side 👍

**Review by zansinergise — APPROVED** [review]
Polygons were created, so map should be available now, good to go! 👍

**Pull request merged**